README.md

```diff
@@ -55,12 +55,23 @@ To run Cirrus tasks, simply switch to a directory where the `.cirrus.yml` is loc
 cirrus run
 ```
 
-It is also possible to run a task by name:
+By default, working directory will be `rsync`ed into a container while respecting `.gitignore`
+configuration. This makes sure Cirrus Tasks are execution from a clean state only with source code
+changes. There is an option to directly mount working directory as is with `--dirty` flag which
+might be useful for "read-only" tasks for purposes like linting or code analysis.
+
+It is also possible to run a particular task by name:
                           
 ```shell script
 cirrus run "Tests (Go 1.15)"
 ```
 
+Or pass some extra environment variables with `-e` flag:
+
+```shell script
+cirrus run -e TEST_PATTER="*Integration*" "Tests (Go 1.15)"
```

Comment:
>`TEST_PATTER`

🤔

It'd probably be better if this will be some real-word example.

Reply:
A suggestion will be appreciated

Reply:
Add an example on testing a release by overriding `CIRRUS_TAG`